packages/esm-home-app/src/homepage-header/homepage-header.component.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Location } from '@carbon/react/icons';
+import { useSession } from '@openmrs/esm-framework';
+import { DatePicker, DatePickerInput } from '@carbon/react';
+import dayjs from 'dayjs';
+import styles from './homepage-header.scss';
+import HomepageIllustration from './homepage-illustration.component';
+
+interface HomepageHeaderProps {
+  title: string;
+}
+
+const HomepageHeader: React.FC<HomepageHeaderProps> = ({ title }) => {
+  const { t } = useTranslation();
+  const session = useSession();
+  const location = session?.sessionLocation?.display;
+
+  return (
+    <div className={styles.header} data-testid="homepage-header">
+      <div className={styles['left-justified-items']}>
+        <HomepageIllustration />
+        <div className={styles['page-labels']}>
+          <p>{t('home', 'Home')}</p>
```

Comment:
Perhaps this could be configurable incase i dont want to call it home or use it it another package